### ISSUE TITLE
Use image input to get results when set relative=True

### ISSUE THREAD

**Issue by ypw1996** [AliaksandrSiarohin/first-order-model#170]
Hello, I'm in great need of your help. How can I get relative results when the input is image.
I have encountered such a problem at present. When the demo input is an image, relative=True is set and the original source image is always obtained. But when the demo input is a video,  it works.
![image](https://user-images.githubusercontent.com/35967597/86212035-eb505900-bba9-11ea-9138-ca1137aa00fd.png)
As in the image, I changed the driver input to the image and I want to use relative. Am I wrong in this way?

**Comment by AliaksandrSiarohin** [comment]
Relative compute motion between first and other frames. So relative need at least 2 frames.

**Comment by ypw1996** [comment]
> Relative compute motion between first and other frames. So relative need at least 2 frames.
Yeah, that's what I was thinking.Can I copy the image that I input and use the same two images as input?

**Comment by AliaksandrSiarohin** [comment]
You will get original image as output in this case, because there is no movement between first and second.

**Comment by ypw1996** [comment]
> You will get original image as output in this case, because there is no movement between first and second.

Ok, I see. Can I crop the video and use keyframes as input?
